trio/_threads.py

```diff
@@ -427,16 +427,18 @@ def from_thread_run_sync(fn, *args, trio_token=None):
     def callback(q, fn, args):
         @disable_ki_protection
         def unprotected_fn():
-            call = fn(*args)
+            ret = fn(*args)
 
-            if inspect.iscoroutine(call):
-                call.close()
+            if inspect.iscoroutine(ret):
+
```

Comment:
See comment above about extra blank lines after block introducers

Reply:
Ups sorry about that, it's burned into my brain